pkg/runtime/interfaces/application_configuration.go

```diff
@@ -78,6 +78,11 @@ type ApplicationConfig struct {
 	Annotations map[string]string `json:"annotations,omitempty"`
 	// Interruptible indicates whether all tasks should be run as interruptible by default (unless specified otherwise via the execution/workflow/task definition)
 	Interruptible bool `json:"interruptible"`
+	// SkipCache indicates all workflows and tasks should skip all their cached results and re-compute their outputs,
+	// overwriting any already stored data.
+	// Note that setting this setting to `true` effectively disabled all caching in Flyte as all executions launched
+	// will have their SkipCache setting enabled.
+	SkipCache bool `json:"skipCache"`
```

Comment:
Is there a known use-case for this? Or just adding for completeness?

Reply:
There is no use-case I could think of that would make sense, however since [`MergeExecutionConfig`](https://github.com/blackshark-ai/flyteadmin/blob/cache-eviction-execution-override/pkg/manager/impl/util/shared.go#L261) is using a [`WorkflowExecutionConfigInterface`](https://github.com/blackshark-ai/flyteadmin/blob/cache-eviction-execution-override/pkg/manager/impl/shared/iface.go#L12) and the application configuration is used as a fallback for other values as well, this was required to satisfy the interface.

It currently returns the actual setting (in case someone wants to set that), however we could also just always have it return `false` for the `ApplicationConfig` and not make it configurable at all on application level.